### PR TITLE
Local database fp63

### PR DESCRIPTION
**Pull request by JosephOri**
## Task link from JIRA:
link: < (example: https://mssplinter10-1711638164183.atlassian.net/jira/software/projects/FP/boards/6?selectedIssue=FP-49)>

## Description:
modify docker-compose in order to run postgres via image from dockerhub

### CODE REVIEW

File: make.sh
Comment (nitzanto) [inline make.sh:1]: docker-compose up -d

should already run the postgres, also the database configuration should be within the docker compose yam file. All of these is not nescessary.

You already have the depend on aswell in the docker compose yaml. So why add a wait command here.

File: docker-compose.yml [hunk 1]
```diff
@@ -31,22 +34,29 @@ services:
     restart: always
     ports:
       - "3001:3000"
+    networks:
```
Comment (nitzanto) [inline docker-compose.yml]: Why did you add the network? This is not necessary

File: .gitignore [hunk 2]
```diff
@@ -2,4 +2,5 @@
 .github/workflows/pr_workflow/tag_code_owners.sh
 .github/workflows/pr_workflow/identify_touched_code.sh
 .github/workflows/pr_workflow/generate_pr_description.sh
-.github/workflows/pr_workflow/create_pr.sh
\ No newline at end of file
+.github/workflows/pr_workflow/create_pr.sh
+node_modules
```
Comment (nitzanto) [inline .gitignore]: Ignore node_modules. Any reason you added that to this PR?

**Review by nitzanto — CHANGES_REQUESTED** [review]

File: docker-compose.yml (hunk 1 above)
Comment (JosephOri) [reply docker-compose.yml]: we need all of the containers to run on the same network

Comment (nitzanto) [reply docker-compose.yml]: @JosephOri They already are..

File: docker-compose.yml [hunk 3]
```diff
@@ -1,3 +1,4 @@
+version: '3.8'
```
Comment (nitzanto) [inline docker-compose.yml]: There's no need to add a version this is deprecated.

**Review by nitzanto — CHANGES_REQUESTED** [review]

**Review by nitzanto — APPROVED** [review]
